README.md

```diff
@@ -409,7 +409,7 @@ Flags:
 1.) **Starting AEMM with `events`**: `events` route available immediately and `spot` routes will no longer be available due to the implementation of Commands [detailed here](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/docs/usage.md):
 
 ```
-$ ec2-metadata-mock events --code instance-reboot -a 2020-01-07T01:03:47Z  -b 2020-01-01T01:03:47Z -l 2020-01-10T01:03:47Z --state completed
+$ ec2-metadata-mock events --code instance-reboot -a 2020-01-07T01:03:47Z  -b 2020-01-01T01:03:47Z -l 2020-01.10.11:03:47Z --state completed
```

Comment:
This looks like an incorrect regex application. Can you revert?

Reply:
The [issue is in `prepare-for-release`](https://github.com/aws/amazon-ec2-metadata-mock/blob/main/scripts/prepare-for-release#L139): the variable`$PREVIOUS_VERSION` does not get escaped before passing to `sed`, so the `.` character is treated as a wildcard rather than a literal period.

Reply:
yeah this happens every few releases; find some unintended value that happens to match that version number pattern.